tasktiger_admin/views.py

```diff
@@ -77,24 +78,39 @@ def task_detail(self, queue, state, task_id):
                 task,
                 execution,
             )
+            execution_converted = convert_keys_to_datetime(
+                execution, ['time_failed', 'time_started']
+            )
             executions_dumped.append(
                 (
-                    json.dumps(execution, indent=2, sort_keys=True),
+                    json.dumps(
+                        execution_converted,
+                        indent=2,
+                        sort_keys=True,
+                        default=str,
+                    ),
                     traceback,
                     execution_integrations,
+                    execution_converted,
                 )
             )
 
         integrations = generate_integrations(
             self.integration_config.get('INTEGRATION_LINKS', []), task, None
         )
 
+        task_data_converted = convert_keys_to_datetime(
+            task.data, ['time_last_queued']
```

Comment:
I would make a `time_last_queued` property on the TaskTiger `Task` class that returns `datetime.datetime.utcfromtimestamp(self._data["time_last_queued"])` and stick to using the `Task` object directly in the template.

Reply:
cool 👍 
https://github.com/closeio/tasktiger/pull/191
Will update this PR's requirements.txt after ^ is released.